bb.edn

```diff
@@ -1,3 +1,15 @@
 {:deps
  {lambdaisland/open-source {:git/url "https://github.com/lambdaisland/open-source"
-                            :git/sha "f2133b2a88bff3898e027c9b50516161c237792e"}}}
+                            :git/sha "f2133b2a88bff3898e027c9b50516161c237792e"}}
+ :tasks
+ {:requires ([babashka.fs :as fs])
+  test:bb {:doc "Run babashka tests with custom runner"
+           :extra-paths ["src" "test"]
+           :extra-deps {current/project {:local/root "."}
+                        org.clojure/test.check    {:mvn/version "1.1.1"}}
+           :task (exec 'lambdaisland.deep-diff2.runner/run-tests)}
+  npm-ws (when-not (fs/exists? "node_modules/ws")
+           (shell "npm install ws"))
+  test:kaocha {:doc "Run kaocha tests"
+               :depends [npm-ws]
+               :task (apply clojure "-M:dev:test -m kaocha.runner" *command-line-args*)}}}
```

Comment:
I would appreciate not adding this to all of our projects, the standard way to run kaocha is through `bin/kaocha`.

Reply:
Done